### ISSUE TITLE
Get simple statistics of purchases

### ISSUE THREAD

**Issue by TTA777** [AnalogIO/shifty-webapp#9]
Needs further discussion with board/ treasurer what would be useful for the to have access to.

Board and maybe some managers will need access.

New endpoint needed

**Comment by TTA777** [comment]
@A-Guldborg Yes, this, along with the other issues, was filed a bit quickly. The need for discussion was mainly regarding what data we would want/ need from the endpoint, since I wasn't aware of the details at the time. 

From what I understand, we have the endpoint now, so no discussion needed for that any more, though it might be worthwhile to discuss how best to show the data to the end-user.

**Comment by A-Guldborg** [comment]
@TTA777 
The endpoint looks as follows:

![image](https://github.com/AnalogIO/shifty-webapp/assets/95026056/405b0ebf-043b-49f8-b1bb-f7a70525ff80)

Basically; A list of products with product id, name, tickets left and the purchase value of the non-redeemed tickets. Data is grouped by productid, meaning if two different users have unused filter coffees but for different purchase prices, they will only show the summed value of all unused filter coffees. The query contains a to and from date, meaning I can query per month, year or any other time-range that I (as the user) decide. I hope this explanation makes sense?

Given this data, I see two options for the data visualization:
- A table-like solution with rows showing the data and a footer summing up the total amount. Something like [this](https://mudblazor.com/components/table).
- A chart-like solution (probably a bar-chart makes the most sense). However, I am not aware of any free solutions providing this easily with razor, free of charge? I have not spent much time looking though, so it might be possible free of charge.

I mean, I don't want to propose the chart-like solution to the treasurer/board if we can't find a free solution. I have also interpreted the use-case/user story somewhat like:
> As a treasurer, I want to be able to determine the total amount of unused clips that we provide support for, in order to determine the amount of money we "owe" to customers.

With this in mind, I think finding a chart might be over-engineering as I see the primary concern as a matter of numbers - not easy visualisation? Of course, we could also expand this in the future to *also* show the data as a chart, so the treasurer easily can convey this information to stakeholders. Likewise, we might also consider if there is an easy way to export this data to a spreadsheet for further calculations etc. This could for instance be a copy-paste solution (copying with comma-seperated values) or downloadable as a `.csv`-file. But let me know your thoughts - should we begin with the table, should we investigate the chart-like solutions, should we focus on the exporting now or maybe later?

**Comment by TTA777** [comment]
I think you are right that a simple table will suffice for now, in regard to visualization. MudBlazor does have rudimentary support for chart e.g. [barcharts](https://www.mudblazor.com/components/barchart), though I think you are right that, the chart wouldn't be enough for their usecase by itself. They need the underlying data easily available as well.

Basically, I agree. Priorities the table view, and play around with the charts if you find it interesting, and have a good idea for how to make it work.
